localisation/pt-br/docs/01-monkey-app.md

```diff
@@ -0,0 +1,164 @@
+    ```
+
+1. Digite o seguinte prompt no GitHub Copilot para gerar um issue no repositório.
+
+    ```text
+    Create a new GitHub issue in my repository titled 'Implement Monkey Console Application' with the following requirements:
+    
+    - Create a C# console app that can list all available monkeys, get details for a specific monkey by name, and pick a random monkey.
+    - The app should use a Monkey model class and include ASCII art for visual appeal.
+    - Add appropriate labels like 'enhancement' and 'good first issue'.
+    - Add some details about how we may go about implementing this and a checklist for what we will need to do.
+    ```
+
+1. Atribua `@Copilot` ao issue e observe o que está acontecendo.
+
+## Iniciar Servidor MCP &ndash; Monkey MCP
+
+1. Abra a Paleta de Comandos digitando `F1` ou `Ctrl`+`Shift`+`P` no Windows ou `Cmd`+`Shift`+`P` no Mac OS, e procure `MCP: List Servers`.
+1. Certifique-se de que o servidor MCP `github` está em execução.
+1. Escolha `monkeymcp` e clique em `Start Server`.
+
+## Desenvolver Aplicativo Monkey com GitHub Copilot e Servidores MCP
+
+1. Digite o seguinte prompt para obter a lista de macacos.
+
+    ```text
+    Get me a list of monkeys that are available and display them in a table with their details.
+    ```
+
+1. Digite o seguinte prompt para obter uma ideia do modelo de dados para um macaco.
+
+    ```text
+    What would a data model look like for this structure?
+    ```
+
+1. Digite o seguinte prompt para criar um arquivo para a classe do modelo de dados.
+
+    ```text
+    Let's create a new file for this class.
+    ```
+
+1. Digite o seguinte prompt para criar uma classe `MonkeyHelper`.
+
+    ```text
+    Let's create a new class called MonkeyHelper that is static. It should manage a collection of monkey data. Include methods to get all monkeys, get a random monkey, find a monkey by name, and track access count to when a random monkey is picked. The data for the monkeys should come from the Monkey MCP server that we just got.
+    ```
+
+1. Digite o seguinte prompt para atualizar o aplicativo de console.
+
+    ```text
+    Let's update the app now to have a nice menu with the following options that will call into that `MonkeyHelper`.
+    
+    1. List all monkeys
+    2. Get details for a specific monkey by name
+    3. Get a random monkey
+    4. Exit app
+
+    Also display some funny ASCII art randomly.
+    ```
+
+1. Digite o seguinte prompt no GitHub Copilot para fazer push do aplicativo de console atualizado.
+
+    ```text
+    Push the current changes to the `mymonkeyapp` branch of the repository.
+    With this branch, create a PR against the `main` branch.
+    Connect this PR to the issue created before.
+    Then, merge this PR and close the issue.
+    ```
+
+---
+
+OK. Você completou o passo "Desenvolvimento de Aplicativo Monkey com MCP". Vamos para o [PASSO 02: Servidor MCP](./02-mcp-server.md).
```

Comment:
Make sure that the disclaimer should be included at the end of the localised document.